pkg/version/version.go

```diff
@@ -22,6 +22,11 @@ type Version struct {
 	Commit string
 }
 
+// Agent returns the Datadog Agent version.
+func Agent() (Version, error) {
+	return New(AgentVersion, Commit)
+}
+
 // New parses a version string like `0.0.0` and a commit identifier and returns a Version instance
 func New(version, commit string) (Version, error) {
 	re := regexp.MustCompile(`(\d+\.\d+\.\d+)(\-[^\+]+)*(\+.+)*`)
```

Comment:
Since it looks like this function is called often, we could move this regex to a global var outside the function so we don't compile every time. MustCompile is meant to be used like that, actually.

Reply:
You're right, let's do that now.